_source/_mhe-nmds-data-entry-tool/2018-2019.md

```diff
@@ -20,6 +20,6 @@ The [user guide (pdf format, 584 kB)][userguide-href] is available for download.
 * [MHE NMDS Data Entry Tool - Interface (zip format, 988 kB)][interface-href]
 * [MHE NMDS Data Entry Tool User Guide (pdf format, 584 kB)][userguide-href]
 
-[interface-href]: /site/assets/files/MHE-NMDS-2018-2019-interface.zip
-[emptydata-href]: /site/assets/files/MHE-NMDS-2018-2019-empty-data.zip
-[userguide-href]: /site/assets/files/MHE-NMDS-2018-2019-DE-Tool-User-Guide.pdf
+[interface-href]: /site/assets/files/1036/MHE-NMDS-2018-2019-interface.zip
+[emptydata-href]: /site/assets/files/1036/MHE-NMDS-2018-2019-empty-data.zip
+[userguide-href]: /site/assets/files/1036/MHE-NMDS-2018-2019-DE-Tool-User-Guide.pdf
```

Comment:
Looks like we have re-introduced meaningless numbered directories we inherited from the old system. Is it needed?

```suggestion
[interface-href]: /site/assets/files/MHE-NMDS-2018-2019-interface.zip
[emptydata-href]: /site/assets/files/MHE-NMDS-2018-2019-empty-data.zip
[userguide-href]: /site/assets/files/MHE-NMDS-2018-2019-DE-Tool-User-Guide.pdf
```